kxml_compiler/parserrelaxng.cpp

```diff
@@ -325,7 +325,7 @@ Schema::Element ParserRelaxng::convertToSchemaElement(Element *e)
     if (e->hasText)
         schemaElement.setText(true);
 
-    foreach (Element *element, e->elements) {
+    for (Element *element : e->elements) {
```

Comment:
```suggestion
    for (Element *element : qAsConst(e->elements)) {
```

(to avoid the detach)

(repeats)